activesupport/lib/active_support/duration/iso8601_parser.rb

```diff
@@ -0,0 +1,122 @@
+require 'strscan'
+
+module ActiveSupport
+  class Duration
+    # Parses a string formatted according to ISO 8601 Duration into the hash.
+    #
+    # See [ISO 8601](http://en.wikipedia.org/wiki/ISO_8601#Durations) for more information.
```

Comment:
So rdoc makes sure the link is proper even if it's followed by a '.' Anyway, lets change to -

`See {ISO 8601}[http://en.wikipedia.org/wiki/ISO_8601#Durations] for more information.`